nodebuilder/tests/swamp/swamp_tx.go

```diff
@@ -19,7 +21,14 @@ func FillBlocks(ctx context.Context, cctx testnode.Context, account string, bsiz
 		time.Sleep(time.Millisecond * 50)
 		var err error
 		for i := 0; i < blocks; i++ {
-			_, err = cctx.FillBlock(bsize, account, flags.BroadcastBlock)
+			var resp *sdk.TxResponse
+			resp, err = cctx.FillBlock(bsize, account, flags.BroadcastSync)
+			if err != nil {
+				break
+			}
+
+			// ensure each tx is included before moving on to avoid account sequence mismatch errors in tests.
+			err = waitForTxResponse(cctx, resp.TxHash, time.Second*1)
```

Comment:
this is another significant change. With `BroadcastBlock` there was no need to wait for anything else, we now need to explicitly wait for the txHash to be included in a block as the `resp` we get back contains the tx hash. We poll until it's included in a block.